packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -1,42 +1,43 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import React from 'react';
-import {
-  participantListStack,
-  participantListStyle,
-  participantListWrapper,
-  listHeader
-} from '../ChatComposite/styles/Chat.styles';
+import { participantListStack, participantListStyle, participantListWrapper, listHeader } from './styles/Chat.styles';
 import {
   OnRenderAvatarCallback,
   ParticipantList,
   ParticipantListProps,
   ParticipantMenuItemsCallback
 } from '@internal/react-components';
-import { FocusZone, Stack } from '@fluentui/react';
-import { useLocale } from '../localization';
-import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
+import { concatStyleSets, FocusZone, Stack, useTheme } from '@fluentui/react';
 import { AvatarPersona, AvatarPersonaDataCallback } from './AvatarPersona';
+import { peopleSubheadingStyle } from './styles/SidePane.styles';
```

Comment:
naming nit: `SidePange.styles` harkens to how this component is used by the `MeetingComposite` (but not `ChatComposite`. How about `ParticipantContainer.styles` :)

Reply:
makes sense to me! will take care of this